test/test-clib-override.cc

```diff
@@ -12,7 +12,7 @@ namespace {
 class CLIBOverrides : public ::testing::Test {
     virtual void SetUp() {
       // Make sure default untagged read mode is set
-      setenv("UNTAGGED_READ_MODE", "AUTO", 1);
+      setenv("__UNTAGGED_READ_MODE", "AUTO", 1);
```

Comment:
was this a bug or does it require a change to the docs?

Reply:
Part of a test case